src/test/scala/com/fulcrumgenomics/testing/UnitSpec.scala

```diff
@@ -49,6 +55,15 @@ trait UnitSpec extends FlatSpec with Matchers {
     val in = SamReaderFactory.make().open(bam.toFile)
     yieldAndThen(in.iterator().toIndexedSeq) { in.safelyClose() }
   }
+
+  /** Generates a command line parser for a class to check that the argument annotations are valid. */
+  protected def checkClpAnnotations[T <: FgBioTool](implicit ct: ClassTag[T], tt: TypeTag[T]): Unit = {
```

Comment:
@nh13 I got tired of [me] screwing up CLP stuff and our tests not catching it.  This probably won't catch everything, but it will at least catch cases where we forget the `clp` annotation or forget `@arg` on params that don't have defaults, or re-use an argument name or flag.

Once this is merged I'll make a sweep and add the test to each our CLP tests.